observatory-platform/observatory/platform/docker/docker-compose.observatory.yml.jinja2

```diff
@@ -307,12 +313,32 @@ services:
     restart: always
     <<: *networks
     depends_on:
-      - postgres
+      postgres:
+        condition: service_healthy
+    healthcheck:
+      test: ["CMD", "nc", "-z", "-v", "apiserver", "{{api_server_port}}"]
+      interval: 5s
+      retries: 5
     command: apiserver
     ports:
       - {{api_server_port}}:{{api_server_port}}
+    entrypoint: /entrypoint-api.sh
   {%- endif %}
 
+  seed_db:
+    build:
+      context: .
+      dockerfile: Dockerfile.seed_db
+      args:
+        - HOST_USER_ID=${HOST_USER_ID}
+    environment:
+      - API_URI={{api_uri}}
```

Comment:
Cleaner to just set `API_URI` environment variable by itself. The api_uri variable is hard coded up the top in the local environment and set from an environment variable in the remote environment.